### ISSUE TITLE
Pobieranie id filtrów z kategorii bez phrase

### ISSUE THREAD

**Issue by bartexs6** [allegro/allegro-api#9468]
Witam,

Czy istnieje możliwość uzyskania id filtrów dla wybranej kategorii bez konieczności podawania phrase (na przykład: https://api.allegro.pl/sale/products?category.id=16242)?

Czy można następnie wyszukiwać produkty wyłącznie za pomocą filtrów, bez podawania phrase?

**Comment by MartaNowaczyk** [comment]
Nie, zgodnie z dokumentacją zasobu [GET /sale/products](https://developer.allegro.pl/documentation/#operation/getSaleProducts), aby skorzystać z dynamicznych filtrów należy najpierw określić frazę lub EAN. Dopiero wtedy możesz uzyskać filtry i z nich skorzystać.

**Comment by bartexs6** [comment]
FIltry z kategorii o id 254260, zwróciły dwa parametry, rozmiar oraz marke. Wpisując filtr marki wszystko działa bez zarzuty, natomiast wpisując filtr rozmiaru nic nie wyszukuje. API zwraca infromację, że id filtru rozmiar jest równe 54:

         "id":"54",
         "name":"Rozmiar",
         "type":"SINGLE",
         
Szukając rozmiaru 34, co z filtru wynosi 2:

            {
               "name":"34",
               "value":"2",
               "idSuffix":null,
               "count":0,
               "selected":false
            },
            
nic nie znajduje - sale/products?phrase=TEST&category.id=254260&54=2, natomiast sale/products?phrase=TEST&category.id=254260&3806=2, znajduje produkty. Wartość parametru o id 54 jest równa 3806_2

**Comment by MartaNowaczyk** [comment]
Gdy podasz niepoprawny filtr zwrócimy wyniki jakbyś go nie użył, stąd zwracamy produkty z różnymi rozmiarami. Jeśli chodzi o brak wyszukiwania rozmiaru 34 to w twoim przykładzie jest "count":0, co by oznaczało, że nie ma produktów z takim parametrem, gdybyś podał trace-id miałabym pewność dla jakich danych było to wyszukiwane.
